bitbot/nick.go

```diff
@@ -0,0 +1,24 @@
+package bitbot
+
+import (
+	"strings"
+
+	"github.com/whyrusleeping/hellabot"
+)
+
+var NickTrigger = NamedTrigger{
```

Comment:
This should be named something more descriptive and specific related to detecting and fixing nick collisions